.github/workflows/golangci-lint.yaml

```diff
@@ -40,7 +40,10 @@ jobs:
           # working-directory: somedir
 
           # Optional: golangci-lint command line arguments.
-          args: -E gosec --timeout=10m
+          args: -E gosec -E gofmt --timeout=10m
+
+          # actions/setup-go already handles caching
+          skip-cache: true
```

Comment:
I added this because I saw tons (like over a thousand) red ERROR lines about how the files golangci-lint was un-tar-ing were getting an error because the files were already present.